### ISSUE TITLE
For NYU, the result is not good

### ISSUE THREAD

**Issue by djx99** [zhyever/Monocular-Depth-Estimation-Toolbox#9]
Referring to BTS and your work, I've got a good result in the kitti dataset, but for NYU, the result is not good.
![00006](https://user-images.githubusercontent.com/86991107/163702846-9aa7e3d3-d934-4fcc-b237-95338a0f78d8.png)
![00006_pre](https://user-images.githubusercontent.com/86991107/163702848-ddfba2d0-d3f5-4dae-b66c-86e2e936c014.png)

I downloaded the NYU Raw Dataset and used project_depth_map.m from the NYU toolkit, to project the depth map on the RGB image plane. dataloader is basically the same as [bts.](https://github.com/cleinc/bts/blob/master/pytorch/bts_dataloader.py) 
Is there any special handling required when doing NYU, or what do you think caused this?

**Comment by zhyever** [comment]
While I did not handle the raw NYU dataset, I think you can refer to this [codes](https://github.com/cleinc/bts/blob/master/utils/extract_official_train_test_set_from_mat.py). In [bts](https://github.com/cleinc/bts), the authors have introduced how to use this python script to convert raw NYU dataset. See **Prepare [NYU Depth V2](https://cs.nyu.edu/~silberman/datasets/nyu_depth_v2.html) test set** in their README.md.

**Comment by zhyever** [comment]
Close for now. Feel free to re-open for further discussion.
